woocommerce-gateway-ebanx/gateways/class-wc-ebanx-global-gateway.php

```diff
@@ -265,18 +269,36 @@ public function init_form_fields() {
 					'11' => '11',
 					'12' => '12',
 				),
-				'description' => __('Establish the maximum number of installments in which your customer can pay, as consented on your contract.', 'woocommerce-gateway-ebanx'),
+				'description' => __('Establish the maximum number of instalments in which your customer can pay, as consented on your contract.', 'woocommerce-gateway-ebanx'),
 				'desc_tip' => true
-			),
+			)
+		);
+		$currency_code = strtolower($this->merchant_currency);
+		if ( in_array(strtoupper($currency_code), WC_EBANX_Constants::$CREDIT_CARD_CURRENCIES) ) {
+			$fields["min_instalment_value_$currency_code"] = array(
+				'title' => sprintf(__('Minimum instalment (%s)', 'woocommerce-gateway-ebanx'), strtoupper($currency_code)),
+				'type' => 'number',
+				'class' => 'ebanx-payments-option',
+				'placeholder' => sprintf(__('The default is %d', 'woocommerce-gateway-ebanx'), 
+					$currency_code === WC_EBANX_Constants::CURRENCY_CODE_MXN ? WC_EBANX_Constants::ACQUIRER_MIN_INSTALMENT_VALUE_MXN : WC_EBANX_Constants::ACQUIRER_MIN_INSTALMENT_VALUE_BRL),
```

Comment:
When currency is COP it will display 20 nonetheless and that's not what we want, is it?
Abstracting this to a method instead of using ternary will be better for setting the default message on this field in a more dynamic way.